e2e_tests/docker_image/parsec-service-test-all.Dockerfile

```diff
@@ -94,15 +94,18 @@ RUN cd nanopb-0.4.4-linux-x86 \
 RUN rm -rf nanopb-0.4.4-linux-x86 nanopb-0.4.4-linux-x86.tar.gz
 
 # Install mock Trusted Services
-RUN git clone https://git.trustedfirmware.org/TS/trusted-services.git --branch main \
+# Setup git config for patching dependencies
+RUN git config --global user.email "some@email.com"
+RUN git config --global user.name "Parsec Team"
```

Comment:
Fro what commands is it needed?

Reply:
Ignore my message, answer [here](https://github.com/parallaxsecond/parsec/pull/406#discussion_r666309798)